src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -804,13 +832,14 @@ public static implicit operator (int a, int b)(NewStruct value)
     }
 }
 ";
-            await TestInRegularAndScriptAsync(text, expected, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+            await Test(text, expected, options: PreferImplicitTypeWithInfo(), testHost: host);
         }
 
         [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.CodeActionsConvertTupleToStruct)]
         public async Task TestNonLiteralNames(TestHost host)
         {
             var text = @"
+using System.Collections.Generic;
```

Comment:
This is out-of-scope for this PR. But I think this `using` should be in the `expected` state even if it doesn't in the original input (i.e, have AddImportsAnnotation on the node for `EqualityComparer`). I personally think the input should remain as-is for this PR, and the behavioral change is addressed in a followup.

Reply:
yes, that's a good point.  i would ok with us making such a behavior change in teh future :)